wasmer/config_opcodes.go

```diff
@@ -0,0 +1,520 @@
+package wasmer
+
+type Opcode int64
```

Comment:
This can probably be `C.wasmer_parser_operator_t` ?

Reply:
agreed